docs/editor/intellisense.md

```diff
@@ -0,0 +1,119 @@
+---
+Order: 4
+Area: editor
+TOCTitle: IntelliSense
+ContentId: 4E9A74AA-D778-4D1C-B369-83763B3C340F
+PageTitle: IntelliSense in Visual Studio Code
+DateApproved: 8/15/2016
+MetaDescription: One of the core features of Visual Studio Code is IntelliSense.  Set breakpoints, step-in, inspect variables and more.
+---
+
+# IntelliSense
+
+IntelliSense is a core feature of Visual Studio Code. When you first download VS Code you will see syntax highlighting for nearly every language, and IntelliSense for JavaScript and TypeScript. 
+
+![IntelliSense demo](images/intellisense/intellisense.gif)
+
+## Language Extensions
+
+IntelliSense for JavaScript and TypeScript is provided out of the box through the [TypeScript](https://github.com/Microsoft/TypeScript/wiki/Salsa) language service. For other languages you will need to [install a language extension](/docs/editor/extension-gallery).
+
+> Tip: For configuring and troubleshooting JavaScript IntelliSense see [here](/docs/languages/javascript#configuring-intellisense).
+
+Below are the most popular language extensions in the [Marketplace](https://marketplace.visualstudio.com/vscode)Click on an extension tile above to read the description and reviews to decide which extension is best for you. 
+
+<div class="marketplace-extensions-languages"></div>
+
+## What is IntelliSense?
+
+IntelliSense is a general term for a variety of code editing features, including: code completion, paramter info, quick info, and list members. 
+
+IntelliSense has been described in a number of ways. 
+
+* IntelliSense is intelligent code completion.
+* IntelliSense is docs at your finger tips
+* IntelliSense provides instant context-aware help. 
+
+In other tools, IntelliSense goes by the terms "code completion", "auto code completion", "content assist", or "code hinting." The IntelliSense experience in VS Code is best-in-class, providing the most intelligent, context-aware suggestions possible. 
+
+## Using IntelliSense
+
+IntelliSense is easy to use. Simply begin typing. While you type the language service will provide code completion and on certain characters (for example `.`) method suggestions will appear. 
+
+> Tip: You can trigger IntelliSense in any editor window by typing <kbd>ctrl+space</kbd>.
+
+In addition, you can see the short documentation (commonly called **quick outline**) for each method (as provided by the language service). 
+
+![quick outline](images/intellisense/quick_outline.png)
+
+After choosing a method you are provided with **paramater info**. 
```

Comment:
parameter
this is misspelled several places. Are you using Sean's spell checking extension?